Azure/sonic-buildimage
# [sonic-py-swsssdk] Update submodule

**- What I did**
Updated sonic-py-swsssdk to upgrade the SDK to be usable for any DB. 
commit info:
"Enhance SDK to read from any DB by name (#49)"
4cee385
https://github.com/Azure/sonic-py-swsssdk/commit/4cee38534919e34f407363ac3ab5f31b4d09be6d

**- How I did it**
Added a db_connect method, that accepts dbname. 
The pre-existing connect() method stays the same (it connects to CONFIG_DB)
The database.json is enhanced to include separator char used in keys, which differs across databases and the code is enhanced to get the separator from the file"

**- How to verify it**
Write a python script to use db_connect for any DB and use methods like get_keys, get_table, ... to verify that it indeed connected to the intended DB.

**- Description for the changelog**

Enhance SDK to read from any DB by name

**- A picture of a cute animal (not mandatory but encouraged)**
